lib/jobs/uefi-job.js

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+
+module.exports = uefiJobFactory;
+di.annotate(uefiJobFactory, new di.Provide('Job.Uefi'));
```

Comment:
@yyscamper We can change the name in the future if we need separate jobs for uefi related workflows.